models/marts/advertising/fct_ad_performance.sql

```diff
@@ -51,15 +51,15 @@ marketing_channels as (
             when lower(campaign) ilike '%display%' then 'display'
             when lower(campaign) ilike '%shopping%'
                 or lower(campaign) ilike '%cpc%' then 'paid search'
-            when utm_medium is null and utm_medium not in ('fb_retargeted', 'ig_retargeted') 
+            when utm_medium is null and utm_campaign not in ('fb_retargeted', 'ig_retargeted') 
                 then 'direct'
             when lower(utm_medium) in ('retargeting', 'social', 'influencers') 
                 then utm_medium
             when lower(utm_campaign) in ('fb_retargeted', 'ig_retargeted') 
                 then 'retargeting'
-            when lower(utm_medium) in ('paid', 'paid_social') 
-                and lower(utm_campaign) not ilike '%amazon%' then 'facebook ads'
             when lower(utm_campaign) ilike '%amazon%' then 'facebook amazon'
+            when lower(utm_medium) in ('paid', 'paid_social')
+                then 'facebook ads'
             when lower(utm_medium) in ('paidsearch', 'cpc', 'shopping') 
                 then 'paid search'
             else null
```

Comment:
@fkalkhalidi a number of ads are showing up with null channel from this categorization - like facebook utm_medium = `prospecting` - is there another way you want these categorized?

Reply:
If there are facebook ads that were being categorized as utm_medium = prospecting then we can categorize that with facebook ads.